templates/Dockerfile.txt

```diff
@@ -1,11 +1,12 @@
-<% if(!coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1<% } %><% if(coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1-coreclr<% } %>
+<% if(!coreclr){ %>FROM microsoft/dotnet<% } %><% if(coreclr){ %>FROM microsoft/dotnet:core<% } %>
 
-RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
-RUN apt-get -qq update && apt-get install -qqy sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
+#Set urls and port environments
+ENV ASPNETCORE_URLS="http://*:5000"
```

Comment:
@peterblazejewicz Alright, this is post-RC2 stuff then! For RC2, "ASPNETCORE_URLS" doesn't work.